language/rescue_spec.rb

```diff
@@ -290,4 +290,11 @@ def rescuer.===(exception)
       :expected
     end.should == :expected
   end
+
+  ruby_version_is "2.4" do
+    it "parses 'rescue' in method arguments" do
```

Comment:
I think `allows` would be more expressive than just `parse`, after all it also has to execute correctly :)